src/components/Modal/VoteModal/VoteModal.jsx

```diff
@@ -1,34 +1,62 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import classNames from "classnames/bind";
 import Button from "components/Button";
 import Image from "components/Image";
 import { addCommas } from "utils/commas";
 import { ReactComponent as XIcon } from "assets/icons/x_icon.svg";
 import femaleData from "mockData/femaleData";
+import useRequest from "hooks/useRequest";
 import DefaultModal from "../Modal";
 import styles from "../Modal.module.scss";
 
-const MOCK_DATA = femaleData.slice(0, 6);
-
 const cn = classNames.bind(styles);
 
-export default function VoteModal({ isOpen, handleModalOpen }) {
+export default function VoteModal({ data, isOpen, handleModalOpen, gender }) {
   const [checkedValue, setCheckedValue] = useState("");
+  const [credit, setCredit] = useState(0);
 
-  const handleRadioChange = (e) => {
-    setCheckedValue(e.target.value);
-  };
+  const getCredit = () => {
+    const storedCredit = localStorage.getItem('Credit');
+
+    return storedCredit || 100000;
+  }
 
   const handleVote = () => {
-    // TODO: 투표 기능 추가
+    const updatedCredit = credit - 1000;
+
+    setCredit(updatedCredit);
```

Comment:
```suggestion
    setCredit((prev) => prev - 1000);
```
원본 state를 setState에서 활용하는 경우 가독성 측면에서 이렇게 사용해보는 것은 어떨까요?

Reply:
`1000`을 변수로 대체하는 것도 좋은 방법일 것 같습니다!